frontend21/scss/components/language-selector.scss

```diff
@@ -0,0 +1,35 @@
+.#{prefix(null, 'language-selector')} {
+  position: relative;
+  display: none;
+  height: 100%;
+
+  @include respond-to('medium') {
+    display: flex;
+  }
+
+  &:after {
+    content: '';
+    position: absolute;
+    top: calc(50% - 4px);
+    right: 5px;
+    width: 8px;
+    height: 8px;
+    background: color('white');
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 64 64' fill='%2320202A'%3E%3Cpath d='M59.863 17.14l4.137 5.488-32 24.232-32-24.232 4.137-5.488 27.863 21.108z'%3E%3C/path%3E%3C/svg%3E");
+    background-repeat: no-repeat;
+    background-position: center;
+    pointer-events: none;
+  }
+
+  select {
+    height: 100%;
+    padding: 0 15px 0 5px;
+    font-family: $fontFamilyRegular;
+    font-size: 16px;
+    font-weight: bold;
+  }
+
+  .mainmenuopen & {
```

Comment:
Is this the only purpose `.mainmenuopen` still exists as a class? If so then I'd vote to always show the language selector on mobile instead.